File: data structures and algorithms/java/05_010.java
import java.util.*;
import java.util.stream.*;
import java.io.*;
import java.math.*;

public class Main {
  public static void main(String[] args) {
     // Main method code goes here
  }
  
  public void stackExample() {
     // Create an empty stack
     Stack<String> stack = new Stack<>();
     // Push items onto stack
     stack.push("first");
     stack.push("second");
     // Pop - returns and removes "second"
     String lastItem = stack.pop();
     // Peek at top item without removing
     String topItem = stack.peek();
     // Check if empty
     boolean isEmpty = stack.isEmpty();
  }
}

File: data structures and algorithms/cs/05_010.cs
using System;
using System.Collections.Generic;

public class MainClass
{
   public static void Main(string[] args)
   {
      // Main method code goes here
   }

   public void StackExample()
   {
      // Create an empty stack
      Stack<string> stack = new Stack<string>();
      // Push items onto stack
      stack.Push("first");
      stack.Push("second");
      // Pop - returns and removes "second"
      string lastItem = stack.Pop();
      // Peek at top item without removing
      string topItem = stack.Peek();
      // Check if empty
      bool isEmpty = stack.Count == 0;
   }

   public void ClearStack(Stack<string> stack)
   {
      stack.Clear();
   }
}
